versioned_docs/version-24.10/monitoring/pluginpacks.md

```diff
@@ -346,15 +346,20 @@ Your Monitoring Connector is now uninstalled.
 
 #### Managing dependencies
 
-You will not be able remove a pack if host and service templates created by the Monitoring Connector are being used by any
-monitored hosts and services.
+If host or service templates provided by the monitoring connector are used by a host, service or template, the monitoring connector cannot be uninstalled.
 
 ![image](../assets/configuration/pluginpacks/uninstall_pp_used.png)
 
-To uninstall the pack you will need to either:
+If the connector is used by a host or service, in order to uninstall it, you will need to:
 
-* delete the hosts and services linked to the templates provided by the Monitoring Connector,
-* or unlink the hosts and services from the corresponding templates.
+* either delete the hosts or services linked to the templates provided by this monitoring connector
+* unlink hosts linked to templates provided by this monitoring connector (note that if this was the only host template applied to this host, it will no longer be monitored).
```

Comment:
```suggestion
* or unlink hosts linked to templates provided by this monitoring connector (note that if this was the only host template applied to this host, it will no longer be monitored).
```